365/client/css/365.css

```diff
@@ -10,29 +10,29 @@ body, html {
 }
 
 /* Header */
-#header button{
+.header button{
   padding:0px;
 }
-#header{
+.header{
   font-family: GeosansLight;
   padding-top: 20px;
   padding-bottom: 20px;
   overflow:hidden;
   background-color:#E8F4FD;
 }
-#header .row > * {
+.header .row > * {
   height: 100px;
 }
-#header .left > * {
+.header .left > * {
   float: left;
 }
-#header .right > * {
+.header .right > * {
   float: right;
 }
-#header .right span {
+.header .right span {
   margin: 0 10px;
 }
-#header .right i {
+.header .right i {
```

Comment:
really sorry but if you can change 'header' back to an id that would be awesome.. also in 365.html. should remove merge conflicts